### PR TITLE
Feature/vehicle base

### PR DESCRIPTION
**Pull request by JoepEilander**

### CODE REVIEW

**Review by lucasvdiepen — CHANGES_REQUESTED** [review]
The wheelbase is not correctly parented to the train cabins.

**Review by lucasvdiepen — APPROVED** [review]

File: Assets/Art/2D/Sprites/S_TrainWheelBig.png
Comment (TryFox) [inline Assets/Art/2D/Sprites/S_TrainWheelBig.png:1]: Add this file to a new folder inside `Sprites` for example `Train`

File: Assets/Art/2D/Sprites/S_TrainWheelSmall.png
Comment (TryFox) [inline Assets/Art/2D/Sprites/S_TrainWheelSmall.png:1]: Add this file to a new folder inside `Sprites` for example `Train`

File: Assets/Art/Materials/Train/M_TrainBase.mat
Comment (TryFox) [inline Assets/Art/Materials/Train/M_TrainBase.mat:1]: Maybe instead of making a new material, use a black spot on another material for the underside

**Review by TryFox — CHANGES_REQUESTED** [review]
Connection part still does not look quite right. Maybe try adding half of the extrusion to both sides? We could also look into parenting objects

File: Assets/Art/Materials/Train/M_TrainBase.mat
Comment (TryFox) [reply Assets/Art/Materials/Train/M_TrainBase.mat:1]: Resolved

**Review by TryFox — CHANGES_REQUESTED** [review]
Same thing as last time, try adding half of the extrusion to both sides. We could also look into parenting objects (sorry for the confusion)

**Review by TryFox — APPROVED** [review]
All changes have been approved

**Review by TryFox — CHANGES_REQUESTED** [review]
There is a scaling issue in the modles, this is an exporting issue. Please fix

**Review by TryFox — APPROVED** [review]

**Pull request merged**